frontend/account.html

```diff
@@ -1,81 +1,111 @@
-    <script
-      src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
-      integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
-      crossorigin="anonymous"
-    ></script>
-    <script
-      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
-      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
-      crossorigin="anonymous"
-    ></script>
-    <script
-      src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
-      integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
-      crossorigin="anonymous"
-    ></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- SEO meta tags -->
+    <meta
+      name="description"
+      content="This is a user account page created by Team PASS: an undergraduate Gemstone research team from UMD studying cybersecurity."
+    />
+    <meta name="author" content="Team Pass" />
+
+    <!-- CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./assets/styles/main.css" type="text/css" />
+
+    <title>Team PASS | User Account</title>
+    <link rel="icon" type="image/png" href="favicon.png" />
+  </head>
+  <body>
+    <main class="container">
+      <img
+        src="./assets/images/team-logo.svg"
+        alt="Team PASS Logo"
+        class="team-logo"
+      />
+      <section class="row justify-content-center">
+        <div class="col-6 card">
+          <div class="card-body">
+            <h3 class="card-title">test@user.com's Account</h3>
+            <p>
+              <strong>Favorite number:</strong> <span id="fav-num">20</span>
+            </p>
+            
+  <!-- Change Made from here -->
+            <div class="form-group">
+              <canvas
+                width="400"
+                height="400"
+                id="snake-game"
+                class="card"
+              ></canvas>
+            </div>
+
+            <button id="btnRevoke" type="button" class="btn btn-primary" value="Revoke_Cred" onclick="fetchBack();">Revoke Credentials</button>
+            
+  <!-- To Here -->
+            <small class="form-text text-muted"
+              >Go back to
+              <a href="./registration.html">registration</a>
+            </small>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- Custom JS (the snake game) -->
+    <script src="./scripts/snake-game.js"></script>
+  <!-- Change Made from here -->
+    <script>

```

Comment:
For organization purposes, it might be useful to put this script into a seperate .js file, like `/scripts/account.js`. That way, we can keep the markup seperate from the logic of the app. Then, instead of setting the `onclick` attribute for the button in the HTML, you can register a click listener instead. That would look something like

```js
const revokeCredsButton = document.getElementById("btnRevoke");

revokeCredsButton.addEventListener("click", fetchBack); // Calls fetchBack whenever the button is pressed
```